packages/@aws-cdk/aws-apigatewayv2/lib/http/integration.ts

```diff
@@ -74,6 +74,60 @@ export class PayloadFormatVersion {
   }
 }
 
+export interface IMappingKey {
+  key: string;
+};
+
+export interface IMappingValue {
+  value: string;
+};
+
+export class MappingKey implements IMappingKey {
+  public static custom(mappingKey: string) {return new MappingKey(mappingKey); }
+  protected constructor(public readonly key: string) {}
+}
```

Comment:
Not clear why we need these classes and interfaces.

Reply:
Removed